omod/src/main/resources/config.xml

```diff
@@ -201,6 +201,12 @@
         <description>Set observation strategy.</description>
     </globalProperty>
 
+	<globalProperty>
+        <property>${project.parent.artifactId}.planDefinition.strategy</property>
```

Comment:
rename this to planDefinition.planDefinitionStrategy to be more consistent

Reply:
Same as above.